Test/WebTestCase.php

```diff
@@ -545,7 +546,7 @@ public function loadFixtureFiles(array $paths = array(), $append = false, $omNam
         $om = $registry->getManager($omName);
 
         if ($append === false) {
-            $this->cleanDatabase($registry, $om);
+            $this->cleanDatabase($registry, $om, $omName);
```

Comment:
I don't know how to retrieve the name `$omName` from `$om`, so I took the argument `$omName` directly.